projects/packages/connection/src/class-client.php

```diff
@@ -316,8 +316,8 @@ public static function _wp_remote_request( $url, $args, $set_fallback = false )
 	/**
 	 * Sets the time difference for correct signature computation.
 	 *
-	 * @param HTTP_Response $response the response object.
-	 * @param Boolean       $force_set whether to force setting the time difference.
+	 * @param array $response Response array from `wp_remote_request`.
```

Comment:
```suggestion
	 * @param array|\WP_Error $response Response array from `wp_remote_request`, or WP_Error on error.
```

Reply:
Yeah, I suppose line 277 might pass that.